docs/ops/memory/mem_migration.md

```diff
@@ -0,0 +1,184 @@
+| taskmanager.memory.preallocate | pre-allocation is no longer supported and managed memory is always allocated lazily, see also [how to migrate managed memory](#managed-memory)                                                                                                                                                          |
+
+The following options are deprecated but if they are still used they will be interpreted as new options for backwards compatibility:
+
+| **Deprecated option**               | **Interpreted as**                                                                                                                                                                                                                                                                                                        |
+| :---------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| taskmanager.heap.size               | - [taskmanager.memory.flink.size](../config.html#taskmanager-memory-flink-size-1) for standalone deployment <br/> - [taskmanager.memory.process.size](../config.html#taskmanager-memory-process-size-1) for containerised deployments <br/> See also [how to migrate total memory](#total-memory-previously-heap-memory). |
+| taskmanager.memory.size             | taskmanager.memory.managed.size, see also [how to migrate managed memory](#managed-memory)                                                                                                                                                                                                                                |
+| taskmanager.network.memory.min      | [taskmanager.memory.network.min](../config.html#taskmanager-memory-network-min)                                                                                                                                                                                                                                           |
+| taskmanager.network.memory.max      | [taskmanager.memory.network.max](../config.html#taskmanager-memory-network-max)                                                                                                                                                                                                                                           |
+| taskmanager.network.memory.fraction | [taskmanager.memory.network.fraction](../config.html#taskmanager-memory-network-fraction)                                                                                                                                                                                                                                 |
+
```

Comment:
These two tables look dazzling after compiled, at least in my browser.
I think it would be better if we draw the lines of the table units.
Shall we consider to use html tables instead of markdown tables?

Reply:
Indeed, I can try this